projects/table-builder/src/lib/classes/table-store.ts

```diff
@@ -20,13 +20,12 @@ export class TableStore extends ComponentStore<TableState> {
    super( { ...defaultTableState, ...config.defaultTableState});
   }
 
-  getSavableState() : Observable<TableState> {
+  getSavableState() : Observable<PersistedTableState> {
     return  this.state$.pipe(
       map( s => {
-        const metaData = Object.values(s.metaData)
-        .map( md => ({...md, transform: undefined }))
-        .reduce((prev: Dictionary<MetaData>, current)=> ({...prev, [current.key]: current}), {});
-        return {...s, metaData };
+        const savableState = {...s}
+        delete savableState.metaData;
```

Comment:
We need a way to persist the sort which currently exists in the metadata.

Reply:
As far as I can tell it is a seperate field on the TableState. The meta data just contains the presort. It is what we don't want to persist. The code can change to define a different presort. Either way the sorted property on the tableState will be properly updated and persisted.